### PR TITLE
corrected missing scope of exception

### PR DESCRIPTION
**Pull request by DarKnight--**

### CODE REVIEW

**Comment by DePierre** [comment]
@DarKnight-- Thanks for catching that up! Could you inline both the lines you changed?

File: framework/db/worklist_manager.py [hunk 1]
```diff
@@ -70,7 +70,7 @@ def _generate_query(self, criteria=None, for_stats=False):
                         criteria['limit'] = criteria['limit'][0]
                     query = query.limit(int(criteria['limit']))
         except ValueError:
-            raise InvalidParameterType(
+            raise exceptions.InvalidParameterType(
```
Comment (delta24) [inline framework/db/worklist_manager.py]: Follow the 120-char limit generously - make it in one line! :)

File: framework/db/worklist_manager.py [hunk 2]
```diff
@@ -59,7 +59,7 @@ def _generate_query(self, criteria=None, for_stats=False):
                         models.Work.target_id.in_(criteria.get('id')))
                 if isinstance(criteria.get('id'), (str, unicode)):
                     query = query.filter_by(
-                        target_id == int(criteria.get('id')))
+                        target_id = int(criteria.get('id')))
```
Comment (delta24) [inline framework/db/worklist_manager.py]: Same here.

**Comment by DePierre** [comment]
@DarKnight-- Thank you very much!

**Comment by DarKnight--** [comment]
@DePierre Happy to help :)

**Pull request merged**